packages/devtools_app/lib/src/framework/release_notes/release_notes.dart

```diff
@@ -23,11 +24,11 @@ bool debugTestReleaseNotes = false;
 // from the flutter/website PR, which has a GitHub action that automatically
 // stages commits to firebase. Example:
 // https://flutter-docs-prod--pr8928-dt-notes-links-b0b33er1.web.app/tools/devtools/release-notes/release-notes-2.24.0-src.md.
-String? _debugReleaseNotesUrl;
+const String? _debugReleaseNotesUrl = null;
 
-const _flutterDocsSite = 'https://docs.flutter.dev';
-
-const releaseNotesKey = Key('release_notes');
+const String _unsupportedPathSyntax = '{{site.url}}';
```

Comment:
nit: replace String with final

Reply:
This variable is already `const`, do you just want me to rely on type inference here?